src/makeRobot.js

```diff
@@ -38,7 +38,51 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    goForward(num = 1) {
```

Comment:
lets rename `num` to `step` all over the code. 
```suggestion
    goForward(step = 1) {
```